### PR TITLE
Replace hardcoded delays with constants from single source, refactor checkForNextStep

### PR DESCRIPTION
**Pull request by Terverok**

### CODE REVIEW

**Comment by Dammic** [comment]
Can one of the admins verify this patch?

File: events/EnactPolicyEvents.mjs [hunk 1]
```diff
@@ -18,6 +18,7 @@ import {
     peekLastEnactedPolicyCard,
     resetFailedElectionsCount,
 } from '../utils/RoomsManager'
+import { TimeDelay } from './consts.mjs'
```
Comment (Dammic) [inline events/EnactPolicyEvents.mjs]: You don't need this .mjs

File: events/EnactPolicyEvents.mjs [hunk 2]
```diff
@@ -61,22 +62,30 @@ export const enactPolicyEvent = (socket, policy) => {
     emits.emitNewPolicy(socket.currentRoom, policy)
 }
 
-export const checkForNextStep = (socket, hasPolicyBeenEnacted = false, customResumeFunc = null) => {
+export const checkForNextStep = (socket, hasPolicyBeenEnacted = false, getCustomNextStep = null, delay = TimeDelay.LONG_DELAY) => {
+    // const additionalDelay = hasPolicyBeenEnacted ? TimeDelay.SHORT_DELAY : 0;
+    // const delay = TimeDelay.LONG_DELAY + additionalDelay;
```
Comment (Dammic) [inline events/EnactPolicyEvents.mjs]: Please fix those comments

File: events/PhaseSocketEvents.mjs [hunk 3]
```diff
@@ -13,6 +13,7 @@ import {
     checkWinConditions,
 } from '../utils/RoomsManager'
 import * as emits from './emits'
+import { TimeDelay } from './consts.mjs'
```
Comment (Dammic) [inline events/PhaseSocketEvents.mjs]: Unnecessary .mjs

File: events/SocketEvents.mjs [hunk 4]
```diff
@@ -69,6 +69,8 @@ import {
 } from '../utils/RoomsManager'
 import * as emits from './emits'
 
+import { TimeDelay } from './consts.mjs'
```
Comment (Dammic) [inline events/SocketEvents.mjs]: Unnecessary mjs

Comment (polarny) [reply events/SocketEvents.mjs]: Done

File: events/PhaseSocketEvents.mjs (hunk 3 above)
Comment (polarny) [reply events/PhaseSocketEvents.mjs]: Done

File: events/EnactPolicyEvents.mjs (hunk 2 above)
Comment (polarny) [reply events/EnactPolicyEvents.mjs]: Done

File: events/EnactPolicyEvents.mjs (hunk 1 above)
Comment (Terverok) [reply events/EnactPolicyEvents.mjs]: done

**Comment by Dammic** [comment]
retest please

**Comment by Dammic** [comment]
retest

**Pull request merged**